airbyte-integrations/connectors/source-surveymonkey/unit_tests/unit_test.py

```diff
@@ -0,0 +1,34 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+from airbyte_cdk.logger import AirbyteLogger
+from source_surveymonkey import SourceSurveymonkey
+
+
+def test_client_wrong_credentials():
```

Comment:
isn't SAT covers this case?

Reply:
we have described integration_test in  readme so I followed it. I have the following entry in acceptance-test-config.yaml.
```  connection:
    - config_path: "secrets/config.json"
      status: "succeed"
    - config_path: "integration_tests/invalid_config.json"
      status: "failed"```
      
    So should I delete this file?

Reply:
I think we don't need this test here, SAT should be good enough